fuzzers/frida_gdiplus/src/fuzzer.rs

```diff
@@ -1,5 +1,11 @@
 //! A libfuzzer-like fuzzer with llmp-multithreading support and restarts
 //! The example harness is built for gdiplus.
+//! NOTE: This file is 1-to-1 copy of the ../../frida-gdiplus/fuzzer.rs, which
```

Comment:
is it a typo here?
`frida_libpng`?

Reply:
oh, sure

Reply:
Idk why `.github/workflows/build_and_test.yml` got into this pull, it supposed to be in another one, but hopefully this will not block the merge

Reply:
sometimes this happens idk why :slightly_frowning_face: